lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -61,6 +61,55 @@ def register_get_thread_response(self, thread):
             status=200
         )
 
+    def register_post_comment_response(self, response_overrides, thread_id=None, parent_id=None):
+        """
+        Register a mock response for POST on the CS comments endpoint for the
+        given thread or parent; exactly one of thread_id and parent_id must be
+        specified.
+        """
+        def callback(request, _uri, headers):
+            """
+            Simulate the comment creation endpoint by returning the provided data
+            along with the data from response_overrides.
+            """
+            response_data = make_minimal_cs_comment(
+                {key: val[0] for key, val in request.parsed_body.items()}
+            )
+            response_data.update(response_overrides)
+            return (200, headers, json.dumps(response_data))
+
+        if thread_id and not parent_id:
+            url = "http://localhost:4567/api/v1/threads/{}/comments".format(thread_id)
```

Comment:
out of curiosity, why not mock the url the same as the actual discussion api 
http://localhost:4567/api/**discussion_api**/v1/threads/{}/comments".format(thread_id)`